cpp/util/CompressionUtil.cpp

```diff
@@ -11,10 +11,22 @@
 #define WINDOW_BIT 15
 #define GZIP_ENCODING 16
 
+#if defined(MSDOS) || defined(OS2) || defined(WIN32) || defined(__CYGWIN__)

```

Comment:
We don't support MSDOS, OS2 and __CYGWIN__

Reply:
Will remove it.